src/core/surface/completion_queue.c

```diff
@@ -283,6 +289,7 @@ grpc_event grpc_completion_queue_pluck(grpc_completion_queue *cc, void *tag,
   ret = ev->base;
   gpr_free(ev);
   GRPC_SURFACE_TRACE_RETURNED_EVENT(cc, &ret);
```

Comment:
It's worthwhile. There are further changes in #1577, and I want someone to do a thorough rewrite of this module entirely soon to eliminate any event related allocations - so maybe leave it til then?